src/renderer/widgets/Image.cpp

```diff
@@ -54,6 +150,8 @@ bool CImage::draw(const SRenderData& data) {
         borderBox.round();
         imageFB.alloc(borderBox.w, borderBox.h, true);
         g_pRenderer->pushFb(imageFB.m_iFb);
+        glClearColor(0.0, 0.0, 0.0, 0.0);
+        glClear(GL_COLOR_BUFFER_BIT);
```

Comment:
can this help with #256? 
though I can't repro, pretty sure it has smth to do with framebuffer
maybe something more needed?

Reply:
ah yea it should